shared/connection.go

```diff
@@ -158,7 +171,22 @@ func (c *Connection) Exec(command string, args ...string) ([]byte, error) {
 
 	cmdArgs := []string{"exec", c.podName}
 	if cmd == "kubectl" {
-		cmdArgs = append(cmdArgs, "-c", "uyuni", "--")
+		namespace := c.namespace
+		if namespace == "" {
+			var namespaceErr error
+			namespace, namespaceErr = kubernetes.GetNamespace(c.kubernetesFilter)
```

Comment:
You surely want to store the found namespace in the connection object...
```suggestion
			namespace, namespaceErr = kubernetes.GetNamespace(c.kubernetesFilter)
			c.namespace = namespace
```

Reply:
Got you! Followed the same approach as [GetPodname](https://github.com/uyuni-project/uyuni-tools/pull/352/files#diff-f9195857a277d58e69d73592278875495daf2ddeacd42678cd2e64033b89fcc8L113-R158)

Reply:
> Got you! Followed the same approach as [GetPodName()](https://github.com/uyuni-project/uyuni-tools/pull/352/files#diff-f9195857a277d58e69d73592278875495daf2ddeacd42678cd2e64033b89fcc8L113-R158)

Yes, but `GetPodName()` stores the value in its field too :wink: